addons/web/static/src/xml/base.xml

```diff
@@ -194,10 +194,10 @@
             <t t-if="state.deletedFavorite.userId">Are you sure that you want to remove this filter?</t>
             <t t-else="">This filter is global and will be removed for everybody if you continue.</t>
             <t t-set="buttons">
-                <button type="button" class="btn btn-primary" t-on-click="_onRemoveFavorite">
+                <button type="button" class="btn btn-primary o_click_dont_close_dropdown" t-on-click="_onRemoveFavorite">
```

Comment:
When it comes to such behaviours it generaly applies to all of the items of a same dropdown and giving a specific className to each individual item might be bothersome. I would rather have a flag in the form of a prop on the dropdown menu itself ("`closeOnSelect`" or something similar) defaulted to `false` and explicitely set to `true` for the action and print menus for example (or the other way around if we find more occurences of the latter scenario).